eng/pipelines/templates/stages/vmr-build.yml

```diff
@@ -373,17 +373,19 @@ stages:
           image: ${{ variables.ubuntuContainerImage }}
         targetOS: linux
         targetArchitecture: x64
-
-    - template: ../jobs/vmr-build.yml
-      parameters:
-        buildName: Windows_BuildTests
-        isBuiltFromVmr: ${{ parameters.isBuiltFromVmr }}
-        vmrBranch: ${{ variables.VmrBranch }}
-        pool: ${{ parameters.pool_Windows }}
-        targetOS: windows
-        targetArchitecture: x64
         extraProperties: /p:DotNetBuildTests=true
```

Comment:
Consider updating the job name to make it clear that this job now builds tests

Reply:
Fixed with https://github.com/dotnet/sdk/pull/47106/commits/936d376568de57f92c765380919e714c519cec71